components.html

```diff
@@ -0,0 +1,946 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Components - Nature UI</title>
+		<link
+			rel="stylesheet"
+			href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+		/>
+		<!-- <link
+			rel="stylesheet"
+			href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.css"
+		/> -->
+
+		<link rel="icon" href="./gardenin-icon.png" type="image/x-icon" />
+		<link href="styles.css" type="text/css" rel="stylesheet" />
+		<link rel="stylesheet"
+      href="//cdn.jsdelivr.net/gh/highlightjs/cdn-release@10.7.2/build/styles/default.min.css">
+<script src="//cdn.jsdelivr.net/gh/highlightjs/cdn-release@10.7.2/build/highlight.min.js"></script>
```

Comment:
Try to add script files at the end of the page, so that after loading the page, not all script files are loaded, it will help in page performance.

Reply:
Yeah, you're right. Just thought if I put it at the end then the page may break if it fails to load the script.

Reply:
Btw done with these changes.